src/models/time-sync.ts

```diff
@@ -20,14 +20,15 @@ const {
   INVALID_TIME_INTERVAL,
   INVALID_OBSERVATION_OVERLAP,
   AVOIDING_PADDING,
+  AVOIDING_PADDING_BY_EDGES: AVOIDING_PADDING_PER_RECEIPT,
```

Comment:
I think we can introduce it with the same name in both places, renaming is unecessary

Reply:
true. fixed